src/commands/synthetics/README.md

```diff
@@ -145,18 +145,25 @@ const { results, summary } = await synthetics.executeTests(...)
 
 ### Use a proxy
 
-It is possible to configure a proxy to be used for outgoing connections to Datadog using the `proxy` key of the global configuration file.
+It is possible to configure a proxy to be used for outgoing connections to Datadog. You can do this by using the `proxy` key of the global configuration file or the `HTTPS_PROXY` environment variable.
+
+⚠️**Note:** This is the only exception where the the global configuration file takes precedence over environment variable and there is no option to set this through the CLI.
 
 As the [`proxy-agent` library][2] is used to configure the proxy, the supported protocols include `http`, `https`, `socks`, `socks4`, `socks4a`, `socks5`, `socks5h`, `pac+data`, `pac+file`, `pac+ftp`, `pac+http`, and `pac+https`. The `proxy` key of the global configuration file is passed to a new `proxy-agent` instance, which means the same configuration for the library is supported.
 
-**Note**: `host` and `port` keys are mandatory arguments and the `protocol` key defaults to `http` if not defined.
+To be able to use a proxy you'll first need to set the CA certificate so datadog-ci can trust your proxy. You can do this by setting the `NODE_EXTRA_CA_CERTS` environment variable to the path of your CA certificate. Otherwise, you might get a `unable to verify the first certificate` error.
```

Comment:
```suggestion
To use a proxy, you need to first set the CA certificate so datadog-ci trusts your proxy. You can do this by setting the `NODE_EXTRA_CA_CERTS` environment variable to the path of your CA certificate. Otherwise, you might get a `unable to verify the first certificate` error.
```